checkov/terraform/checks/resource/azure/MLComputeClusterMinNodes.py

```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+
+class MLComputeClusterMinNodes(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure Machine Learning Compute Cluster Minimum Nodes Set To 0"
+        id = "CKV_AZURE_150"
+        supported_resources = ['machine_learning_compute_cluster']
```

Comment:
```suggestion
        supported_resources = ['azurerm_machine_learning_compute_cluster']
```
I think you meant this resource name 🙂